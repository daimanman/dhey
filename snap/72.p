{"C":1,"Duration":185856884,"EndTime":1537232485727459700,"Err":"","FileId":"3","InputFileName":"file","Method":"POST","N":1,"P":null,"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":1537232485541602800,"Status":1,"TaskId":72,"Type":"FILE","Url":"http://192.168.1.192:9091/api/v1/oss","Z":""}