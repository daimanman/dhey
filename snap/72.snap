{"AvgTotal":0.1858569,"Fastest":0.1858569,"Slowest":0.1858569,"Average":0.1858569,"Rps":5.380483587103842,"AvgConn":0,"AvgDNS":0,"AvgReq":0,"AvgRes":0.0031996,"AvgDelay":0.1816876,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.0031996,"DelayMax":0.1816876,"DelayMin":0.1816876,"Lats":[0.1858569],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0],"ResLats":[0.0031996],"DelayLats":[0.1816876],"Total":185856900,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":0,"SizeReq":0,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.1858569,"Count":1,"Frequentcy":1},{"Mark":0.1858569,"Count":0,"Frequentcy":0},{"Mark":0.1858569,"Count":0,"Frequentcy":0},{"Mark":0.1858569,"Count":0,"Frequentcy":0},{"Mark":0.1858569,"Count":0,"Frequentcy":0},{"Mark":0.1858569,"Count":0,"Frequentcy":0},{"Mark":0.1858569,"Count":0,"Frequentcy":0},{"Mark":0.1858569,"Count":0,"Frequentcy":0},{"Mark":0.1858569,"Count":0,"Frequentcy":0},{"Mark":0.1858569,"Count":0,"Frequentcy":0},{"Mark":0.1858569,"Count":0,"Frequentcy":0}],"TaskId":72,"Duration":0.1858569}